regular-expression.dd

```diff
@@ -134,20 +139,19 @@ $(D_S Regular Expressions,
     )
     ---
     auto text = readText(...);
-    auto replaced = replace(text,
-        regex(r"([0-9]{1,2})/([0-9]{1,2})/([0-9]{4})","g"), "$(DOLLAR)3-$(DOLLAR)1-$(DOLLAR)2");
+    auto replaced = replaceAll(text, r"([0-9]{1,2})/([0-9]{1,2})/([0-9]{4})".regex, "$3-$1-$2");
     ---
-    $(P The "all dates" part comes from the fact that regex has global option set, omit it to replace only the first occurrence.
-    As seen the replacement is controlled by format string not unlike one in C's famous printf.
-    The &#36;1, &#36;2, &#36;3 substituted with content of sub-expresions.
+    $(P The $(D "pattern".regex) is just another notation of writing $(D regex("pattern")) called UFCS that some
```

Comment:
UFCS is now in the glossary. It's not too important, but you can link to the definition from here.